src/tutorial/modal.component.test.tsx

```diff
@@ -41,16 +54,33 @@ describe('TutorialModal', () => {
   test('sends correct data to the root component when walkthrough button is clicked', async () => {
     const user = userEvent.setup();
 
-    render(<TutorialModal open={true} onClose={jest.fn()}/>);
+    render(<TutorialModal open={true} onClose={jest.fn()} />);
+
+    const walkthroughButton = screen.getByText('Basic Tutorial').closest('.tutorialItem').querySelector('.walkthrough');
 
-    const walkthroughButton = screen.getByText('Walkthrough');
     await user.click(walkthroughButton);
 
     expect(setSteps).toHaveBeenCalledWith(mockTutorialData[0].steps);
     expect(setShowTutorial).toHaveBeenCalledWith(true);
   });
 
-  test.todo('renders tutorials properly');
+  test('renders tutorials properly', async () => {
+    const user = userEvent.setup();
+    render(<TutorialModal open={true} onClose={jest.fn()} />);
+
+    const walkthroughButton = screen.getByText('Basic Tutorial').closest('.tutorialItem').querySelector('.walkthrough');
+    await user.click(walkthroughButton);
 
+    await waitFor(() => {
+      mockTutorialData[0].steps.forEach((step) => {
+        const element = screen.queryByText(step.content);
+        if (element) {
+          console.log('Found element:', element);
+        } else {
+          console.log('Element not found for:', step.content);
+        }
```

Comment:
No need to use console logs here. You can use an assertion.

Reply:
@Vijaykv5